Src/AutoFixture.NUnit3/AutoDataAttribute.cs

```diff
@@ -64,12 +64,18 @@ private TestCaseParameters GetParametersForMethod(IMethodInfo method)
             }
         }
 
-        private IEnumerable<object> GetParameterValues(IEnumerable<IParameterInfo> parameters)
```

Comment:
FWIW, this change is related to what @hackle commented [here](https://github.com/AutoFixture/AutoFixture/pull/589#issuecomment-202308310).